.github/scripts/build_assets/arg_getters.py

```diff
@@ -36,7 +36,7 @@ def get_selenium_runner_args(peek_mode=False):
     if peek_mode:
         parser.add_argument("pr_title",
                             help="The title of the PR that we are peeking at")
-    else:
+    if has_token != False:
```

Comment:
This is a bit easier on the eyes imo :)
```suggestion
    if not has_token:
```

It's preference ofc, so feel free to ignore this if you disagree